.env.development

```diff
@@ -9,6 +9,7 @@ POSTGRES_NAME=laa_crime_application_store
 
 # To disable need for authentication locally
 # AUTHENTICATION_REQUIRED=false
+REDIS_URL=redis://localhost:6379/1
```

Comment:
local setup has the submit application using `redis://localhost:6379/1` [as here](https://github.com/ministryofjustice/laa-submit-crime-forms/blob/main/config/initializers/sidekiq.rb#L31) which i think could cause "job confusion" for sidekiq if we use it here. I think, in any event, it gets overridden in the sidekiq [initializer here](https://github.com/ministryofjustice/laa-crime-application-store/blob/main/config/initializers/sidekiq.rb#L19-L26).

Reply:
cheers - will have another look at this